src/main/java/jenkins/branch/WorkspaceLocatorImpl.java

```diff
@@ -233,6 +233,14 @@ private static FilePath locate(TopLevelItem item, String fullName, Node node, bo
         return map;
     }
 
+    // Workaround spotbugs false-positive redunant null checking for try blocks in java 11
+    // https://github.com/spotbugs/spotbugs/issues/756
+    private static @NonNull InputStream readFilePath(@NonNull FilePath index) throws IOException, InterruptedException {
```

Comment:
We do not use Glick-style annotations. If the method returns `@NonNull` then the `@NonNull` shall be on the line preceding the access modifiers not between the access modifiers and the return type